bukkit/src/main/java/com/github/games647/fastlogin/bukkit/listener/protocollib/VerifyResponseTask.java

```diff
@@ -162,8 +165,8 @@ private void verifyResponse(BukkitLoginSession session) {
             } else {
                 //user tried to fake an authentication
                 disconnect("invalid-session",
-                    "GameProfile {0} ({1}) tried to log in with an invalid session ServerId: {2}",
-                    session.getRequestUsername(), socketAddress, serverId);
+                    String.format("GameProfile %s (%s) tried to log in with an invalid session. ServerId: %s",
```

Comment:
SLF4J uses {} for logging, instead of {numeric} which was missing in the migration to it. Maybe it's better to use the formatting of the logger than to do it manually.

Reply:
Yeah this is just a left-over from my debugging since SLF4J would not for some reason expand the parameters and I wanted to see them so I just put this in place. I will put it back as it was.

Reply:
Resolved in https://github.com/games647/FastLogin/pull/811/commits/6413ca4d109d942d2c461916263413218b0551b4